lib/action_controller/serialization.rb

```diff
@@ -18,34 +19,25 @@ def serialization_scope
         respond_to?(_serialization_scope, true)
     end
 
+    # Deprecated
     def get_serializer(resource)
-      @_serializer ||= @_serializer_opts.delete(:serializer)
-      @_serializer ||= ActiveModel::Serializer.serializer_for(resource)
-
-      if @_serializer_opts.key?(:each_serializer)
-        @_serializer_opts[:serializer] = @_serializer_opts.delete(:each_serializer)
-      end
-
-      @_serializer
+      fail ActiveModel::Serializer::Error, "ActionController::Serialization##{__method__} has been removed. "\
+        "Please use ActiveSupport::Serializer#build instead"
     end
```

Comment:
Thought this was better than a straight-up deprecation just in case anyone is using these methods. But this won't handle people that override the methods and expect them to have some effect.  I'm not sure the best way to handle that.